### ISSUE TITLE
old-division regression (not yet released)

### ISSUE THREAD

**Issue by asottile** [PyCQA/pylint#2887]
This is a result of the fix for #2808 

### Steps to reproduce

```python
a / int(b)
```

### pylint 2.3.1

```console
$ pylint --py3k t.py
************* Module t
t.py:1:0: W1619: division w/o __future__ statement (old-division)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### master behaviour

```console
$ pylint --py3k t.py

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 0.00/10, +10.00)
```

### pylint --version output

```console
$ pylint --version
pylint 2.4.0-dev0
astroid 2.2.5
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0]

$ git rev-parse HEAD
377e82fdf21c74fe497217bfc97f696b36ead233
```

**Comment by PCManticore** [comment]
Thanks for the report @asottile !

**Comment by asottile** [comment]
I have a patch ~almost ready -- should be able to PR tomorrow
